src/olmo_core/data/numpy_dataset.py

```diff
@@ -503,6 +506,156 @@ def _get_file_size_and_length(self, path, dtype=None) -> Tuple[int, int]:
             raise RuntimeError("invalid 'max_target_sequence_length' or 'sequence_length'")
 
 
+class NumpyFSLDatasetMixture(NumpyFSLDataset):
+    """
+    A version of :class:`NumpyFSLDataset` built from a mixture of sources and their expected token ratios relative to each other. A path_offset_index is used to determine the number of instances to retain from a path when constructing the local indices.
```

Comment:
This would render poorly in the docs without the double backticks:
```suggestion
    A version of :class:`NumpyFSLDataset` built from a mixture of sources and their expected token ratios relative to each other. A ``path_offset_index`` is used to determine the number of instances to retain from a path when constructing the local indices.
```

Reply:
BTW you can build and serve the docs locally by running `make docs`.